### PR TITLE
Add image treatment CSS to Carousel

### PR DESCRIPTION
**Pull request by austingardner**
## Context
Adding option to Carousel at add recent image treatment CSS changes.

## References
https://github.com/eBay/skin/issues/1281

## Screenshots
Screen Shot 2020-12-08 at 12 59 28 PM

### Note that the third image in the gif poking out of its container is an optical illusion; feel free to run it and check for yourself that it's working correctly. I couldn't get the gif not to show that. In reality it is contained within its container.
image-treatment-ebay-ui-2

### CODE REVIEW

File: src/components/ebay-carousel/README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Name | Type | Stateful | Required | Description
 `index` | String | Yes | No | 0-based index position
 `items-per-slide` | String | No | No | automatically fit a number of items for each carousel slide and enable slide controls. If set to a whole number, will default to x.1 where x is the whole number set.
 `gap` | String | No | No | override the margin between carousel items in pixels (default: "16")
+`image-treatment` | Boolean | No | No | If true, image treatment styles are applied. Default is false.
```
Comment (ianmcburnie) [inline src/components/ebay-carousel/README.md:26]: I think an enumerated type would give more room to grow for other potential image treatments. So, `image-treatment=none` by default, then maybe `image-treatment=matte`by default?

Comment (ianmcburnie) [reply src/components/ebay-carousel/README.md:26]: If we like the term `matte`, then we could also maybe use it in our skin classname...?

Comment (austingardner) [reply src/components/ebay-carousel/README.md:26]: Yeah we could do that... but I mean... YAGNI? 😁

Comment (ianmcburnie) [reply src/components/ebay-carousel/README.md:26]: Oh, I don't know. Maybe an `image-treatment=warhol` easter egg? Or `image-treatment=cubism`... if we get all artsy and highbrow 😁

Comment (austingardner) [reply src/components/ebay-carousel/README.md]: Haha sorry about having to google YAGNI. 

I updated image-treatment to be an enum 👍

**Review by ianmcburnie — APPROVED** [review]

**Review by agliga — APPROVED** [review]

**Pull request merged**